lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -18,8 +18,11 @@ all_interfaces:
   multiple:
   ios_xr:
     get_command: 'show running all-interfaces'
+    get_value: '/^interface (.*)/'
+  nexus:
+    get_command: "show running-config interface | section '^interface'"
+    get_value: '/(.*)/'
   get_context: ~
-  get_value: '/^interface (.*)/'
```

Comment:
@mikewiebe  `section` causes some issues for some providers; e.g. `interface_service_vni`. See my other comment below.

Reply:
@mikewiebe per IRC I'm going to add a separate `all_interfaces` getter specific to vni. That seems to be the only provider having issues.